app/models/viewmodels/Decoder.scala

```diff
@@ -27,11 +27,9 @@ class Decoder {
 
   def soe(code: String): String = soeMappings(code)
 
-  private val crcMappings: Map[String, String] = Map(
-    "000" -> "Success",
-    "101" -> "Pre-lodged Declaration has not arrived ",
-    "102" -> "Declaration has not arrived"
-  ).withDefaultValue("")
+  private val crcMappings: Map[String, String] =
+    Map("000" -> "Success", "101" -> "Pre-lodged Declaration has not arrived ", "102" -> "Declaration has not arrived")
```

Comment:
@PatrykRudnicki huh... so ugly...

Reply:
ScalaFormatter, I added failing when code is not formatted